ufw/files/default/ufw.default.tmpl.jinja

```diff
@@ -1,17 +1,15 @@
-{% set ufw_cfg = pillar.get('ufw', {}) -%}
-{% set settings_cfg = ufw_cfg.get('settings', {}) -%}
-{% set ipv6 = "yes" if settings_cfg.get('ipv6', True) else "no" -%}
-{% set default_input_policy = settings_cfg.get('default_input_policy', 'DROP') -%}
-{% set default_output_policy = settings_cfg.get('default_output_policy', 'ACCEPT') -%}
-{% set default_forward_policy = settings_cfg.get('default_forward_policy', 'DROP') -%}
-{% set default_application_policy = settings_cfg.get('default_application_policy', 'SKIP') -%}
-{% set manage_builtins = "yes" if settings_cfg.get('manage_builtins', False) else "no" -%}
-{% set ipt_sysctl = settings_cfg.get('ipt_sysctl', '/etc/ufw/sysctl.conf') -%}
-{% set ipt_modules = settings_cfg.get('ipt_modules', ['nf_conntrack_ftp', 'nf_nat_ftp', 'nf_conntrack_netbios_ns'])|join(" ") -%}
-
-# /etc/default/ufw
-#
-# File managed by Salt. Do not edit manually.
+########################################################################
+# File managed by Salt at <{{ source }}>.
+# Your changes will be overwritten.
+########################################################################
+{%- set ipv6 = "yes" if config.get('ipv6', True) else "no" %}
```

Comment:
It may be too late to change this now (not backwards compatible) but couldn't this be improved here:

https://github.com/saltstack-formulas/ufw-formula/blob/16169a5cf7334576848c313534430e81d01a8b73/pillar.example#L7

Instead setting `"yes"` or `"no"` there would mean this `if` conditional would be unnecessary.  And the default to `True`/`"yes"` could be set in `defaults.yaml`.

Perhaps if this PR ends up as a `BREAKING CHANGE`.